src/test/java/ibm/jceplus/junit/openjceplusfips/TestAESCopySafe.java

```diff
@@ -39,7 +39,7 @@ public static void main(String[] args) throws Exception {
     //
     //
     public static Test suite() {
-        TestSuite suite = new TestSuite(TestAESGCMCopySafe.class);
+        TestSuite suite = new TestSuite(TestAESCopySafe.class);
```

Comment:
Should we remove this method and `main` as well?

Reply:
Same comment/question as above.